pandas/_libs/hashtable_class_helper.pxi.in

```diff
@@ -652,10 +697,14 @@ cdef class {{name}}HashTable(HashTable):
             Unique values of input, not sorted
         labels : ndarray[intp_t] (if return_inverse)
             The labels from values to uniques
+        mask : ndarray[bool], optional
+            If not None, the mask is used as indicator for missing values
+            (True = missing, False = valid) instead of `na_value` or
```

Comment:
Was this meant to be added to the Parameters section? 
(although actually also the return value changes if passing a mask, maybe that can be clarified as well)

Reply:
Yes thanks. Moved the mask part up and added a result_mask return value